app/src/test/java/sdp/moneyrun/RiddleTest.java

```diff
@@ -0,0 +1,94 @@
+package sdp.moneyrun;

```

Comment:
Most of this class is redundant, the Riddle tests are currently in the GameTest file and are already implemented so you should remove any redundant test.

Reply:
I see that you deleted the other class, I guess that's fine, but I think it would be best to use the Assert throws instead of a try-catch, you can just copy-paste whats was there previously

Reply:
I don't see how these tests are redundant, and I don't see how they are already implemented in GameTest, can you give me an example? 

For the try catch, I understand where you're coming from but it's just a minor implementation detail. It could be changed but I don't see it as necessary

Reply:
Well, you removed the whole class where the tests were, not sure why, but everything was already tested.

Reply:
Before, the Tests for the Riddle class were in the GameTest, this was because at the very start the Riddle class was originally defined inside the same file as the Game class. There was no reason for Riddle to not have it's own file like any other class. As a consequence it didn't make sense for the Riddle tests to stay inside the GameTest file as it can only lead to confusion, the RiddleTest file was then created. They are no longer tested in any GameTest class.

Reply:
Also, since Game went through heavy restructuring, and its tests were changed to the Instrumented Tests folder, in order to have access to the DatabaseReference, the GameTest class has been renamed to GameInstrumentedTest

Reply:
Also I added both equals and hashcode to Riddle, so new tests have indeed been added to cover both thsoe methods.

Reply:
I understand, I just didn't see the need to delete them and recode them, you could have copy-pasted them. Regardless, code coverage is 100% so it does not matter, just maybe go back to the assert throws instead of the try-catch, it seems more concise and clear.